diesel_cli/README.md

```diff
@@ -4,6 +4,34 @@ Diesel CLI
 Diesel CLI is a tool that aids in managing your database schema. Migrations are
 bi-directional changes to your database that get applied sequentially.
 
+Installation
+---------------
+
+The diesel cli, by default, requires [`libpq`][postgres], [`sqlite`][sqlite]
+and [`mysql`][mysql], some of which may already be included on your system. See
+which dependencies are already present with:
+
+```sh
+which psql
+which sqlite3
+which mysql
```

Comment:
<img width="540" alt="screen shot 2017-07-29 at 4 22 42 pm" src="https://user-images.githubusercontent.com/1529387/28747989-2ee497a0-747a-11e7-9c6a-df9cf5294c31.png">

I don't think there's any command we can give here which will work in a cross-platform way. Do you think we should we just focus on the Diesel CLI specific stuff and assume that folks can figure out whether they have something installed on their own? (Worst case, attempting to install diesel_cli will tell them)